src/api.ts

```diff
@@ -8,7 +8,7 @@ export default async function createApi() {
   const WS_URL = process.env.WS_URL || 'ws://127.0.0.1:9944'
 
   // explicitely state what RPC we are connecting to
-  console.log('Connecting to RPC at: ' + WS_URL)
+  console.error('Connecting to RPC at: ' + WS_URL)
```

Comment:
This is only an informational message that was introduced to have certainty about what node I'm connecting to. Why is there a change to `.error()`?

Reply:
so it logs to stderr and not stdout. This way when you pipe the output of the script to a .json file you don't need to edit it and remove this line :)